services/web-server/src/servers/createApp.js

```diff
@@ -12,6 +12,8 @@ const credentials = require('./credentials');
 module.exports = async ({ cfg, strategies }) => {
   const app = express();
 
+  // trust first proxy
+  app.set('trust proxy', 1);
```

Comment:
It looks like this unconditionally trusts the proxy?  Did you mean to use `cfg.trustProxy` here?

Reply:
Indeed. Just pushed a fix :)